auto-claude-ui/src/renderer/lib/markdown-task-parser.ts

```
@@ -0,0 +1,300 @@
+
+    // Parse content based on current section
+    switch (currentSection) {
+      case 'description':
+        descriptionLines.push(line);
+        break;
+
+      case 'subtasks':
+        // Check if this line starts a new subtask
+        // IMPORTANT: Only H3/H4 headers are treated as new subtasks
+        // Bullets and numbered lists under headers are part of the description
+        const subtaskTitle = extractSubtaskTitleFromHeader(line);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)